priv/repo/migrations/20220910110436_add_oban_producers.exs

```diff
@@ -0,0 +1,5 @@
+defmodule Notesclub.Repo.Migrations.AddObanProducers do
+  use Ecto.Migration
+
+  defdelegate change, to: Oban.Pro.Migrations.Producers
```

Comment:
Check whether that Pro migration is available:

```elixir
  def change do
    if Code.ensure_loaded?(Oban.Pro.Migrations.Producers) do
      Oban.Pro.Migrations.Producers.change()
    else
      :ignored
    end
  end
```

Reply:
Added, thanks!